docs/installation-configuration.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Aiven Provider for Pulumi Installation & Configuration
+meta_desc: Provides an overview on how to configure the Pulumi Aiven Provider for Pulumi.
+layout: package
+---
+## Installation
+
+The aiven provider is available as a package in all Pulumi languages:
+
+* JavaScript/TypeScript: [`@pulumi/aiven`](https://www.npmjs.com/package/@pulumi/aiven)
+* Python: [`pulumi-aiven`](https://pypi.org/project/pulumi-aiven/)
+* Go: [`github.com/pulumi/pulumi-aiven/sdk/v6/go/aiven`](https://github.com/pulumi/pulumi-aiven)
+* .NET: [`Pulumi.Aiven`](https://www.nuget.org/packages/Pulumi.Aiven)
+* Java: [`com.pulumi/aiven`](https://central.sonatype.com/artifact/com.pulumi/aiven)
+
+The Pulumi provider for [Aiven](https://aiven.io/), the trusted open source data platform for everyone.
+## Authentication
+Sign up for Aiven and [create a personal token](https://aiven.io/docs/platform/howto/create_authentication_token).
+
+You can also create an [application user](https://aiven.io/docs/platform/howto/manage-application-users) and use its token for accessing the Aiven Provider.
+## Example usage
+
+{{< chooser language "typescript,python,go,csharp,java,yaml" >}}
+{{% choosable language typescript %}}
+```yaml
+# Pulumi.yaml provider configuration file
+name: configuration-example
+runtime: nodejs
+config:
+    aiven:apiToken:
+        value: 'TODO: var.aiven_api_token'
+
+```
+
+{{% /choosable %}}
+{{% choosable language python %}}
```

Comment:
haha, it's funny that since this is all config, the content under every language is actually yaml

Reply:
Not quite - the `language` field is unique. Most providers have more code here.